website/docs/r/redis_cache.html.markdown

```diff
@@ -65,6 +65,8 @@ The following arguments are supported:
 
 * `private_static_ip_address` - (Optional) The Static IP Address to assign to the Redis Cache when hosted inside the Virtual Network. Changing this forces a new resource to be created.
 
+* `public_network_access_enabled` - (Optional) Whether or not public network access is allowed for this Redis Cache. Defaults to `true`.
```

Comment:
Might be best to expand on what public network access means
```suggestion
* `public_network_access_enabled` - (Optional) Whether or not public network access to the internet is allowed for this Redis Cache. Defaults to `true`.
```

Reply:
Updated